### DISCUSSION TITLE
SIEM + Endpoint Security not seen

### DISCUSSION THREAD

**Discussion by Cosmin98-sys** [Security-Onion-Solutions/securityonion#1582] [General] [answered]
Hi! I want to use "SIEM + Endpoint Security" part from Kibana because I need to automate detections. When I want to do that in the panel is written that I have to enable a module in filebeat for Suricata, but it is already enabled. I can do to start this facility?

Thank you !

**Comment by weslambert** [comment]
We distributed Security Onion with the open source version of the Elastic Stack.  To enable use of the non-OSS version/basic license, you'll want to consider using `so-features-enable`

**Comment by Cosmin98-sys** [comment, marked as answer]
I did this but all I receive is : 
Downloading so-elasticsearch:'2.3.0'-features
invalid reference format
Error parsing reference: "securityonion/so-elasticsearch:'2.3.0'-features" is not a valid repository/tag: invalid reference format
invalid reference format
Downloading so-filebeat:'2.3.0'-features
invalid reference format
Error parsing reference: "securityonion/so-filebeat:'2.3.0'-features" is not a valid repository/tag: invalid reference format
invalid reference format
Downloading so-kibana:'2.3.0'-features
invalid reference format
Error parsing reference: "securityonion/so-kibana:'2.3.0'-features" is not a valid repository/tag: invalid reference format
invalid reference format
Downloading so-logstash:'2.3.0'-features
invalid reference format
Error parsing reference: "securityonion/so-logstash:'2.3.0'-features" is not a valid repository/tag: invalid reference format
invalid reference format

**Reply by TOoSmOotH** [reply]
This was fixed in 2.3.1. Just run soup on the manager.

**Reply by Cosmin98-sys** [reply]
I did it! But now I still cannot use it because, as I can see here, I'm not de admin..... Any advice of what I can do ?
